extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -38,5 +67,53 @@ describes.realWin(
       expect(() => shoppingConfig.layoutCallback()).to.not.throw();
       expect(shoppingConfig.isLayoutSupported(Layout.NODISPLAY)).to.be.true;
     });
+
+    it('throws on no config', async () => {
+      expectAsyncConsoleError(async () => {
+        expect(async () => {
+          await shoppingConfig.getConfig();
+        }).to.throw(
+          /The amp-story-auto-ads:config should be inside a <script> tag with type=\"application\/json\"​​​/
+        );
+      });
+    });
+
+    it('does use remote config when src attribute is provided', async () => {
+      const exampleURL = 'foo.example';
+      const xhrService = Services.xhrFor(win);
+      const fetchStub = env.sandbox.stub(xhrService, 'fetchJson').resolves({
+        json: () => Promise.resolve(configData),
+      });
+      element.setAttribute('src', exampleURL);
+
+      const result = await shoppingConfig.getConfig();
+      expect(fetchStub).to.be.calledWith(exampleURL);
+      expect(result).to.eql(configData);
+    });
+
+    it('does use inline config when remote src is invalid', async () => {
+      const exampleURL = 'invalidRemoteURL';
+      element.setAttribute('src', exampleURL);
+
+      shoppingConfig.getConfig().then((storyConfig) => {
+        //if remote config is not valid, it will use the inline config
```

Comment:
Nit: `//`-style comments usually begin with a space and then a capital letter:

`// If remote config is not valid, it will use the inline config`

Reply:
Sounds good, had to remove the comments as I am now using a public facing API call rather than using the get config method(s) directly.